src/blog/2023/09/dashboard-notebook-layout.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Dynamic Markdown, Tables & Notebooks with Dashboard 2.0
```

Comment:
Yeah, thats a question for @MarianRaphael really, but seems reasonable. My only thinking is that "FlowFuse Dashboard" feels like something _in_ FlowFuse, given we're a 1-product company right now, rather than something that's a Node-RED node set.